graphql-database-manager-test/src/main/java/com/fleetpin/graphql/database/manager/test/HistoryProcessor.java

```diff
@@ -61,8 +61,13 @@ public void process() {
 		        	
 		        	var items = new HashMap<String, List<WriteRequest>>();
 		        	var writeRequests = stream.map(item -> WriteRequest.builder().putRequest(builder -> builder.item(item)).build()).collect(Collectors.toList());
-		        	items.put(table+"_history", writeRequests);
-		        	client.batchWriteItem(builder -> builder.requestItems(items)).get();
+		        	if (writeRequests == null || writeRequests.isEmpty()) {
```

Comment:
would not have an empty if with an else would just change the if to 

`writeRequests != null && !writeRequests.isEmpty()`